### DISCUSSION TITLE
Symmetry, perspective, isometric, and 2D Grid

### DISCUSSION THREAD

**Discussion by Madflame-Creations** [keiwando/toonsquid#467] [Ideas]
Hello! Can you pls add four features: symmetry, perspective, isometric, and 2D Grid? 
This is a beneficial. Procreate has those features.

The guide options for symmetry are vertical, horizontal, quadrant, and radial.

Thank you for making this app exist :).

**Comment by kevincreative** [comment]
I second this request especially symmetry. It seems most drawing apps have some form of it these days. Iso grid would be a bonus too.

**Comment by Lioncub901** [comment]
This would be cool

**Comment by budaloops** [comment]
Animatable perspective and warp is a feature planned for the next major update, so you won’t have to wait too long for that one. Here is the list of upcoming features here https://toonsquid.com/updates/ToonSquid-2-Beta/

**Reply by Lioncub901** [reply]
Oh not the animatable perspective but a perspective grid to help draw things in perspective like a house or box (just like Procreate and Clip Studio). I mainly want symmetry.

**Reply by budaloops** [reply]
Ohh gotcha

**Comment by Lioncub901** [comment]
very useful. not really the perspertive but the symmetry, isometric, and 2d grid.
